types/chrome/index.d.ts

```diff
@@ -5551,12 +5551,12 @@ declare namespace chrome {
             /** Optional. Time-to-live of the message in seconds. If it is not possible to send the message within that time, an onSendError event will be raised. A time-to-live of 0 indicates that the message should be sent immediately or fail if it's not possible. The maximum and a default value of time-to-live is 86400 seconds (1 day). */
             timeToLive?: number | undefined;
             /** Message data to send to the server. Case-insensitive goog. and google, as well as case-sensitive collapse_key are disallowed as key prefixes. Sum of all key/value pairs should not exceed gcm.MAX_MESSAGE_SIZE. */
-            data: Object;
+            data: object;
         }
 
         export interface IncomingMessage {
             /** The message data. */
-            data: Object;
+            data: object;
```

Comment:
These data objects seem to be JSON objects, which again, would be better typed as key-unknown.